src/Lykke.Service.BitcoinGold.API.AzureRepositories/Binder/RepositoryModule.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Autofac;
+using AzureStorage.Blob;
+using AzureStorage.Tables;
+using Common.Log;
+using Lykke.Service.BitcoinGold.API.AzureRepositories.Asset;
+using Lykke.Service.BitcoinGold.API.AzureRepositories.Operations;
+using Lykke.Service.BitcoinGold.API.AzureRepositories.SpentOutputs;
+using Lykke.Service.BitcoinGold.API.AzureRepositories.Transactions;
+using Lykke.Service.BitcoinGold.API.AzureRepositories.Wallet;
+using Lykke.Service.BitcoinGold.API.Core.Asset;
+using Lykke.Service.BitcoinGold.API.Core.ObservableOperation;
+using Lykke.Service.BitcoinGold.API.Core.Operation;
+using Lykke.Service.BitcoinGold.API.Core.Settings.ServiceSettings;
+using Lykke.Service.BitcoinGold.API.Core.TransactionOutputs;
+using Lykke.Service.BitcoinGold.API.Core.Transactions;
+using Lykke.Service.BitcoinGold.API.Core.Wallet;
+using Lykke.SettingsReader;
+
+namespace Lykke.Service.BitcoinGold.API.AzureRepositories.Binder
+{
+    public class RepositoryModule : Module
+    {
+        private readonly ILog _log;
+        private readonly IReloadingManager<BitcoinGoldApiSettings> _settings;
+        public RepositoryModule(IReloadingManager<BitcoinGoldApiSettings> settings, ILog log)
+        {
+            _log = log;
+            _settings = settings;
+        }
+
+        protected override void Load(ContainerBuilder builder)
+        {
+            RegisterRepo(builder);
+            RegisterBlob(builder);
+        }
+
+        private void RegisterRepo(ContainerBuilder builder)
+        {
+            builder.RegisterInstance(new AssetRepository())
+                .As<IAssetRepository>();
+
+            builder.RegisterInstance(new OperationMetaRepository(
+                AzureTableStorage<OperationMetaEntity>.Create(_settings.Nested(p => p.Db.DataConnString),
```

Comment:
It's better to add static Create method to the repository and incapsulate table name and other repo-creation logic inside repo